### ISSUE TITLE
[FEATURE] There should be a way to see what version you are currently running from within the app

### ISSUE THREAD

**Issue by jlar0che** [HemmeligOrg/Hemmelig.app#328]
### Description

First, thanks for this amazing application!

I run Hemmelig in a Docker container and upgrade whenever there is a new version, but that process is a bit complicated because there is no way to see the current installed version from within the application's web interface. Currently, I keep a note about the version I have installed as a comment within my docker-compose.yml and update it once I upgrade the version I'm running. But, being able to see it in the app would be very useful!

Perhaps I don't see it and this is already the case?

**Comment by bjarneo** [comment]
Absolutely. If you see the admin panel, the navigation menu at the bottom is the location where the version will be. Or, at least, my thought of where it should be.

Anyway, I agree without! SHA commit ID and the GitHub tag.

**Comment by jlar0che** [comment]
Thanks so much for adding this simple, but very useful feature! You're the best!
